### ISSUE TITLE
Decline reason preview box bug

### ISSUE THREAD

**Issue by NovemLinguae** [WPAFC/afch-rewrite#147]
Steps to reproduce:

- Go to a random draft
- Hit decline
- Pick a reason
- Hit preview
- Pick a second reason
- The preview box will say something like "bio,v" instead of the reason previews

**Comment by suryagowda** [comment]
I found no issues in this project afch-rewrite.Are all the issues closed?

**Comment by NovemLinguae** [comment]
> I found no issues in this project afch-rewrite.Are all the issues closed?

No. I see 57 open issues.

**Comment by suryagowda** [comment]
yes.. sorry, But when i open the issue... There was no code. I can see only few comments

**Comment by enterprisey** [comment]
@suryagowda we have plenty, and you're commenting on one right now! feel free to take a look at [this list of issues for newcomers](https://github.com/WPAFC/afch-rewrite/issues?q=is%3Aissue+is%3Aopen+label%3AE-good-first-issue) or [our developer guide](https://en.wikipedia.org/wiki/Wikipedia:WikiProject_Articles_for_creation/Helper_script/Contributing).

...I would certainly be happier if we had no open issues :)
